src/libraries/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/RandomNumberGeneratorImplementation.Windows.cs

```diff
@@ -7,6 +7,15 @@ namespace System.Security.Cryptography
 {
     internal sealed partial class RandomNumberGeneratorImplementation
     {
+        // a singleton which always calls into a thread-safe implementation
+        // and whose Dispose method no-ops
+        internal static readonly RandomNumberGeneratorImplementation s_singleton = new RandomNumberGeneratorImplementation();
```

Comment:
Did you mean to put this in the shared partial in RandomNumberGeneratorImplementation.cs?

Reply:
Wouldn't be the most embarrassing mistake I've made this week. Will retcon this next week.